storage/client/queries/queries.go

```diff
@@ -170,6 +170,20 @@ const (
 		LIMIT $2::bigint
 		OFFSET $3::bigint`
 
+	DelegationsTo = `
+		-- delegatee_info is used to calculate the escrow amount of the delegators in base units
+		WITH delegatee_info AS (
+			SELECT escrow_balance_active, escrow_total_shares_active
+			FROM chain.accounts
+			WHERE address = $1::text
+		)
+	        SELECT delegator, shares, delegatee_info.escrow_balance_active, delegatee_info.escrow_total_shares_active
+			FROM chain.delegations, delegatee_info
+			WHERE delegatee = $1::text
+		ORDER BY delegator, shares
```

Comment:
Kinda silly, but `delegator, shares DESC` is probably more natural? Or even `shares DESC, delegator`. Until we have user-controlled sorting in the UI, this is the order they're bound to want to see.

Reply:
Agreed that `shares DESC, delegator` makes most sense. Also updated for `Delegations` query.